gradle/versions.gradle

```diff
@@ -172,9 +174,7 @@ dependencyManagement {
 
     dependency 'org.web3j:abi:4.8.9'
     dependency 'org.web3j:besu:4.8.9'
-    dependency('org.web3j:core:4.8.9') {
-      exclude group: 'com.github.jnr', name: 'jnr-unixsocket'
-    }
+    dependency 'org.web3j:core:4.8.9'
```

Comment:
we did exclude this to avoid a CVE

Reply:
Oh, ok. I thought it was because of licensing issues. I can revert that but I'll have to remove the web3j test along with it.

Reply:
It seems to be [fixed](https://github.com/web3j/web3j/issues/1616) in `org.web3j:core:4.9.1`. I can try submitting another PR upgrading the version and then rebase this one on top of the new `main`, wdyt?

Reply:
sounds good to me - tag @antonydenyer for review

Reply:
Given that we only use web3j in tests and we're actually using ipc now the risk is very low. That said I'd attempt an upgrade as part of this pr, looking through the changes in web3j they look minimal.

Reply:
@antonydenyer I gave a try to upgrade web3j to `4.9.0` (not `4.9.1` b/c there is not such version for the Gradle plugin) in #3752. Beside of some failing tests, this ends up in some classpath cluttering, as far as I can see in IntelliJ, and I would like to take time to investigate.
By removing the unit test and adding an analogous one into the acceptance test scope, the usage of this library is again isolated. So, if this is acceptable, I would suggest to not block this PR b/c of the re-inclusion of the `jnr-unixsocket` dependency and continue with the review.

Reply:
Agreed - it's a bit of a mess!